src/routes/DigitalAssets/routes/EditAsset/index.js

```diff
@@ -1,9 +1,16 @@
 // @flow
 
+import { ga } from 'utils/analytics'
 import EditAssetContainer from './EditAssetContainer'
 
 const EditAsset = {
   path: 'edit-asset/:assetAddress',
+  onEnter: (nextState: ReactRouterState): void =>
+    ga(
+      'send',
+      'pageview',
+      nextState.location.pathname
+    ),
```

Comment:
would be better if we have some function that hide `'send', 'pageview'` params, to avoid passing them each call.

For example
```
import { sendPageView } from 'utils/analytics'

import EditAssetContainer from './EditAssetContainer'

const EditAsset = {
  path: 'edit-asset/:address',
  onEnter: (nextState: ReactRouterState): void => sendPageView(nextState.location.pathname),
  component: EditAssetContainer,
}
```

Reply:
also this allow to hide what specific service we use - `ga`, etc

Reply:
Hiding which service we use under facade is actually a bad idea.

In short, every analytics service has its own model for events / views / parameters / etc. I was designing such facade for Sberbank and we ended up with our own analytical model that mapped to every analytical service in use. This task was enormously complex and costed a lot of time.

If you would like to hear more about that, ask me in person please. :)

Reply:
Upgraded `ga` to `gaSendPageView` for those places